VERSION

```diff
@@ -1 +1 @@
-2.5.0
\ No newline at end of file
+2.6.0-rc.0
```

Comment:
Uhm... this should be 2.6.0.

Reply:
Have we released 2.6.0 stable with the -rc.0 version inside it?

Reply:
No, this seems like a problem with just this PR: https://github.com/grafana/mimir/blob/release-2.6/VERSION

Reply:
```
$ ./mimir-linux-amd64 -version
Mimir, version 2.6.0 (branch: release-2.6, revision: 27698f399)
  go version:       go1.19.3
  platform:         linux/amd64
```